### ISSUE TITLE
New version available

### ISSUE THREAD

**Issue by josvo** [guillaumedsde/docker-scanservjs#1]
Hi

Thanks for this project - exactly what I needed to get my old scanner working! There is a new version of scanservjs available (2.1.1 at the time of writing). Maybe you could push an updated docker image?
I tried to build it myself, however I always run in an npm error. 
Thanks!

**Comment by guillaumedsde** [comment]
hi, I noticed the CI broke, I'll self assign this and fix it when I have some time :)

**Comment by guillaumedsde** [comment]
Hi @josvo I pushed a fix to the development branch, once [this pipeline](https://gitlab.com/guillaumedsde/docker-scanservjs/-/pipelines/219521541) finishes could you pull the `development` tag from the docker hub and let me know if it is fixed for you?

**Comment by guillaumedsde** [comment]
actually, once [this pipeline](https://gitlab.com/guillaumedsde/docker-scanservjs/-/pipelines/219527160) finishes

**Comment by josvo** [comment]
The pipeline seems to have failed again.

**Comment by guillaumedsde** [comment]
yeah, I thought it was fixed but it looks like the gitlab runners aren't suited for these types of long tasks,  I'll see what I can do about it when I have some time again, probably this weekend

**Comment by digital-grease** [comment]
Hey, just wondering if there's been any update on this?
